build.linux32x86/editpharoinstall.sh

```diff
@@ -11,8 +11,11 @@ if [ "$1" = -source ]; then
 else
 	SourceFile="PharoV50"
 fi
-SOURCE=../../sources/$SourceFile.sources
-test -f $SOURCE || SOURCE=../../../sources/$SourceFile.sources
```

Comment:
As always, history. _sources_ is a well-known idea in the Smalltalk world, and "imageSource" actually does not cut it, as it is not the source from which images are built.

Btw: The Image looks for the sources file in several locations, also depending on platform.
Eg, for Mac it looks:
 * in the current working directory
 * in the directory of the `.image` file
 * in the `Resources` folder of the App bundle
 * in the directory the App bundle resides in
 * (maybe something I forgot)

On Linux, is definitely also multiple locations…